src/WCSimConstructRealisticPlacement.cc

```diff
@@ -660,9 +660,11 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructRealisticPlacement()
     multipmt_central_rotation->rotateZ(270*deg);
 
     // For the OD PMT we instead need to make it face outwards from the white tyvek in the OD.
+    // Rotating with respect to ID PMT orientation
     G4ThreeVector odpmt_central_position = G4ThreeVector(config.WhiteTyvekOuterRadius+2*cm,0.0,0.0);
-    G4RotationMatrix* pmtod_central_rotation = new G4RotationMatrix;
-    pmtod_central_rotation->rotateY(270*deg);
+    G4RotationMatrix* pmtod_central_rotation = pmt_central_rotation;
```

Comment:
Oh wait... it's probably this
This is basically assigning an alias to the ID barrel rotation matrix, then rotating it - which will effect both ID & OD
```suggestion
    G4RotationMatrix* pmtod_central_rotation = new G4RotationMatrix(*pmt_central_rotation);
```